Content.Server/Holosign/HolosignProjectorComponent.cs

```diff
@@ -0,0 +1,24 @@
+namespace Content.Server.Holosign
+{
+    [RegisterComponent]
+    public sealed class HolosignProjectorComponent : Component
+    {
+        [ViewVariables]
+        [DataField("charges")]
+        public int Charges = 6;
+
+        [ViewVariables(VVAccess.ReadWrite)]
+        public int CurrentCharges = 6;
+
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("signProto")]
+        public string SignProto = "HolosignWetFloor";
+
+        [ViewVariables(VVAccess.ReadWrite)]
+        public float Accumulator = 0f;
+
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("rechargeTime")]
+        public TimeSpan RechargeTime = TimeSpan.FromSeconds(30);

```

Comment:
If this doesn't have ItemStatus then there's no need for an accumulator because you can infer charges based on:
- how many there are
- how long it takes to accumulate one
- when it was last used